radio/util/tts_de.py

```diff
@@ -11,31 +11,31 @@
 for i in range(101):
     systemSounds.append((str(i), filename(PROMPT_SYSTEM_BASE + i)))
 systemSounds.append(("tausend", filename(PROMPT_SYSTEM_BASE + 101)))
-for i, s in enumerate(["comma", "und", "minus", "uhr", "minute", "minuten", "sekunde", "sekunden"]):
+for i, s in enumerate(["komma", "und", "minus", "uhr", "minute", "minuten", "sekunde", "sekunden"]):
     systemSounds.append((s, filename(PROMPT_SYSTEM_BASE + 102 + i)))
 for i, (s, f) in enumerate([(u"Volt","volt0"),
                             (u"Ampere", "amp0"),
-                            (u"MilliAmpere", "mamp0"),
+                            (u"Milliampere", "mamp0"),
                             (u"Knoten", "knot0"),
                             (u"Meter pro sekunde", "mps0"),
                             (u"Fuss pro sekunde", "fps0"),
                             (u"Kilometer pro stunde", "kph0"),
-                            (u"Meilen pro Stunde", "mph0"),
+                            (u"Meile pro Stunde", "mph0"), (u"Meilen pro Stunde", "mph1"),
```

Comment:
Adding plural values will not make them available. Currently, the code dealing with German only deals with plural for hours. If you need plural, then this need to be changes and all plural for units needs to be provided.

Also, you need to try those with PicoTTS under linux to listen to what is said, the way it sounds not always match proper spelling